packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -332,4 +336,9 @@ export class DefaultCrudRepository<T extends Entity, ID>
   protected toEntities(models: juggler.PersistedModel[]): T[] {
     return models.map(m => this.toEntity(m));
   }
+
+  async migrateSchema(options?: SchemaMigrationOptions): Promise<void> {
```

Comment:
Same here - `automigrate` and `autoupdate`.  At repository level, maybe it's better to keep the names as `automigrate` and `autoupdate`.